pkg/experimentalmetricmetadata/entity_events.go

```diff
@@ -18,6 +20,7 @@ const (
 
 	semconvOtelEntityID         = "otel.entity.id"
 	semconvOtelEntityType       = "otel.entity.type"
+	semconvOtelEntityInterval   = "otel.entity.interval"
```

Comment:
Please also update the design doc here to keep up to date: https://docs.google.com/document/d/1Tg18sIck3Nakxtd3TFFcIjrmRO_0GLMdHXylVqBQmJA/edit#heading=h.1kzy7lkx5qw1